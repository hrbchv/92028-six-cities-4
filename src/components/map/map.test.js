import React from "react";
import renderer from "react-test-renderer";
import Map from "./map.jsx";

jest.mock(`../map/map.jsx`, () => `Map`);

const mockProps = {
  variantsOfRent: [{
    name: `Wood and stone place`,
    kind: `Room`,
    rating: 3,
    cost: 189,
    imageUrl: `img/apartment-02.jpg`,
    type: `Standart`,
    isBookMark: false,
    coordinates: [123441, 3423423]
  }]
};

it(`Render Map`, () => {
  const tree = renderer
    .create(<Map
      variantsOfRent={mockProps.variantsOfRent}/>, {
      createNodeMock: () => document.createElement(`div`)
    })
    .toJSON();

  expect(tree).toMatchSnapshot();
});
