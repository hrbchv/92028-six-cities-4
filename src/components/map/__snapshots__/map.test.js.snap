// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Map 1`] = `
<Map
  variantsOfRent={
    Array [
      Object {
        "coordinates": Array [
          123441,
          3423423,
        ],
        "cost": 189,
        "imageUrl": "img/apartment-02.jpg",
        "isBookMark": false,
        "kind": "Room",
        "name": "Wood and stone place",
        "rating": 3,
        "type": "Standart",
      },
    ]
  }
/>
`;
